commons/src/main/kotlin/com/simplemobiletools/commons/activities/BaseSimpleActivity.kt

```diff
@@ -95,7 +95,7 @@ abstract class BaseSimpleActivity : AppCompatActivity() {
 
         super.onCreate(savedInstanceState)
         if (!packageName.startsWith("com.simplemobiletools.", true)) {
-            if ((0..50).random() == 10 || baseConfig.appRunCount % 100 == 0) {
+            if ((0..50).random() == 10 || this.baseConfig.appRunCount % 100 == 0) {
```

Comment:
no need to complicate code with "this", it creates tons of changes for no reason